puppet/modules/logfile/manifests/init.pp

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
i propose to move this define to site_config/manifests/logfile, because imo puppet/modules/ should hold general purpose modules that are abstracted from project-specific needs (which should go in site_* modules), and which could ideally made public.
This define is clearly project specific (hardcoded /var/log/leap i.e.)
But i'm open for a discussion on this !